# Request: Only requesting one thing, so no need for a request variable

---
# Response: A vector of string commands for the supervisor to execute
uint32 tagID
int8 status